charts/kong/example-values/minimal-kong-standalone.yaml

```diff
@@ -0,0 +1,19 @@
+# The minimum values.yaml configuration necessary for Kong without the ingress controller, using the Postgres subchart
+image:
+  repository: kong
+  tag: "2.0"
+
+env:
+  prefix: /kong_prefix/
+  database: postgres
+
+postgresql:
+  enabled: true
+  postgresqlUsername: kong
+  postgresqlDatabase: kong
+  service:
+    port: 5432
+
```

Comment:
Do you not need to deploy an "admin" service in this case? Else, how are you supposed to configure Kong?

Reply:
Yeah, good catch--this and minimal-k4k8s-with-kong-enterprise.yaml need it (the latter to make Manager useful). However, they're in kind of a weird place since the services aren't exactly accessible unless we turn them into LoadBalancer types. Debating whether to do that since it will usually be non-free, but simpler, or instruct people to use port-forwards. `KONG_ADMIN_API_URI` is a bit of an issue also, since we can't rely on ingress auto-populating it.

Reply:
Went with port-forwards. This conveniently lets us ignore `KONG_ADMIN_API_URI` and `KONG_ADMIN_GUI_URL`, since they're all localhost!